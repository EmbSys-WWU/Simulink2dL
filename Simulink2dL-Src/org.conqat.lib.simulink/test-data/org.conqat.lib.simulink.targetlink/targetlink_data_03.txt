struct('comment',struct('text','','placement',1),'logdata',struct('maxnumvals',inf,'plotchannels',1,'loggingmode',1,'channels',1),'output',struct('variable','','description','','name','$S_$B','width',1,'address','','max',97.5345,'min',0,'class','default','type','uint32','scaling','','lsb',1e-05,'offset',0,'arb',1,'freeze',1,'unit','','autoscalingmode',1,'lowerheadroom',1,'upperheadroom',1,'headroomunit','bit','keepdatatype',1,'checkmin',0,'checkmax',0,'castoutput',0,'show',0),'inputs','**','exceptionhandling',1,'scalingvalid',0,'date',733382.50538803241)
