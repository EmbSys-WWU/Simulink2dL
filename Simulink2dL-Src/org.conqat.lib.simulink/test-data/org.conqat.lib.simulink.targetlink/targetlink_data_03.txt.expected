struct (
  'comment' -> struct (
    'placement' -> '1'
    'text' -> ''
  )
  'logdata' -> struct (
    'channels' -> '1'
    'loggingmode' -> '1'
    'maxnumvals' -> 'inf'
    'plotchannels' -> '1'
  )
  'output' -> struct (
    'address' -> ''
    'arb' -> '1'
    'autoscalingmode' -> '1'
    'castoutput' -> '0'
    'checkmax' -> '0'
    'checkmin' -> '0'
    'class' -> 'default'
    'description' -> ''
    'freeze' -> '1'
    'headroomunit' -> 'bit'
    'keepdatatype' -> '1'
    'lowerheadroom' -> '1'
    'lsb' -> '1e-05'
    'max' -> '97.5345'
    'min' -> '0'
    'name' -> '$S_$B'
    'offset' -> '0'
    'scaling' -> ''
    'show' -> '0'
    'type' -> 'uint32'
    'unit' -> ''
    'upperheadroom' -> '1'
    'variable' -> ''
    'width' -> '1'
  )
  'date' -> '733382.50538803241'
  'exceptionhandling' -> '1'
  'inputs' -> '**'
  'scalingvalid' -> '0'
)